AnnotatoSharedLibrary/Sources/AnnotatoSharedLibrary/Models/Document.swift

```diff
@@ -6,9 +6,9 @@ public final class Document: Codable {
     public let ownerId: String
     public let baseFileUrl: String
     public private(set) var annotations: [Annotation]
-    public let createdAt: Date?
-    public let updatedAt: Date?
-    public let deletedAt: Date?
+    public var createdAt: Date?
+    public var updatedAt: Date?
+    public var deletedAt: Date?
```

Comment:
Any reason why they are changed only for `Document` and not the rest?

Reply:
Ok my bad on this I made a mistake.
Just to confirm, when a document is updated, all the annotations and children will also have updated timings to right?

Reply:
Ok done